task/03-date-tasks.js

```diff
@@ -56,7 +58,14 @@ function parseDataFromIso8601(value) {
  *    Date(2015,1,1)    => false
  */
 function isLeapYear(date) {
-   throw new Error('Not implemented');
+   let data = new Date(date);
```

Comment:
please fix block formatting according to google code style  https://google.github.io/styleguide/jsguide.html#formatting-nonempty-blocks